Example/PageBuilderCountdown/view/base/web/js/content-type/example-countdown/appearance/default/widget.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+define([
+    'jquery',
+    'Example_PageBuilderCountdown/js/content-type/example-countdown/countdown'
+], function ($, setCountdownField) {
+    'use strict';
+
+    return function (config, element) {
+        var countdownField = element[0].children[0],
+            intervalId = 0;
+
+        intervalId = setInterval(function (field, intervalId) {
+            let target = '';
+            let countdownField = field[0].children[0];
+
+            if (countdownField) {
+                target = countdownField.getAttribute('target_date');
+            }
+
+            this.setCountdownField(target, countdownField, intervalId);
```

Comment:
Same here, we shouldn't have this in front.

Reply:
this is that weird thing we talked about where it doesn't work without "this." in front of the function name. (I accidentally removed it from the backend file but have restored it.)